src/app/pages/SpecialDetailPage/specials/Animation.jsx

```diff
@@ -82,12 +82,13 @@ const Animation = ({ contentLink, title, results }) => {
           <EditorialContent>
             {intro && <Paragraph strong dangerouslySetInnerHTML={{ __html: intro }} />}
             {body && <CustomHTMLBlock body={body} />}
-            {links?.length &&
-              links.map(({ uri, title: linkTitle }) => (
-                <StyledLink variant="with-chevron" href={uri} title={linkTitle} target="_blank">
-                  {linkTitle}
-                </StyledLink>
-              ))}
+            {links?.length
+              ? links.map(({ uri, title: linkTitle }) => (
+                  <StyledLink variant="with-chevron" href={uri} title={linkTitle} target="_blank">
+                    {linkTitle}
+                  </StyledLink>
+                ))
+              : null}
```

Comment:
This could also have used optional chaining:

```suggestion
            {links?.map(({ uri, title: linkTitle }) => (
                <StyledLink variant="with-chevron" href={uri} title={linkTitle} target="_blank">
                  {linkTitle}
                </StyledLink>
              )}
```